conda/cli/main_list.py

```diff
@@ -79,7 +83,7 @@ def configure_parser(sub_parsers: _SubParsersAction, **kwargs) -> ArgumentParser
         type=comma_separated_stripped,
         dest="list_fields",
         help="Comma-separated list of fields to print. "
-        f"Valid values: {sorted(CONDA_LIST_FIELDS)}.",
+        f"Valid values: {', '.join(sorted(CONDA_LIST_FIELDS))}.",
```

Comment:
space or no space

Reply:
Both work as input (but the space requires quoting, of course). Probably easier without.
```suggestion
        f"Valid values: {','.join(sorted(CONDA_LIST_FIELDS))}.",
```